HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh

```diff
@@ -36,7 +36,7 @@ done
 ALL_FILES="${ALL_FILES%?}"
 echo "Discovered files: $ALL_FILES"
 
-cmsDriver.py Phase2 -s HLT:75e33_timing --processName=HLTX \
+cmsDriver.py Phase2 -s L1P2GT,HLT:75e33_timing --processName=HLTX \
```

Comment:
is there any plan to associate a unit test to this script such that's protected against this sort of updates?

Reply:
Ciao @mmusich,
this script will be run on every IB. If there are errors, we will know.

Reply:
>  this script will be run on every IB. 

by whom?

Reply:
it will be integrated in the usual IB tests that are "centrally run".
A bot, I guess, 😄

Reply:
This is what I mean, currently it's not the case as far as I cant tell. Is there a plan to make it run during IB tests?

Reply:
Yes @mmusich, there is a plan that has been agreed with @smuzaffar and @fwyzard before adding the script to the release.

Reply:
yes , we will add an additional test to run it for IBs and PRs